docs/en/migration.md

```diff
@@ -1 +1,511 @@
+    persistent_workers=False,  # Whether maintain the workers Dataset instances alive
+    sampler=dict(type='InfiniteSampler', shuffle=True),  # The type of data sampler
+    dataset=dict(  # Train dataset config
+        type=dataset_type,  # Type of dataset
+        ann_file='meta_info_DIV2K800sub_GT.txt',  # Path of annotation file
+        metainfo=dict(dataset_type='div2k', task_name='sisr'),
+        data_root=data_root + '/DIV2K',  # Root path of data
+        data_prefix=dict(  # Prefix of image path
+            img='DIV2K_train_LR_bicubic/X2_sub', gt='DIV2K_train_HR_sub'),
+        filename_tmpl=dict(img='{}', gt='{}'),  # Filename template
+        pipeline=train_pipeline))
+val_dataloader = dict(
+    num_workers=4,  # The number of workers to pre-fetch data for each single GPU
+    persistent_workers=False,  # Whether maintain the workers Dataset instances alive
+    drop_last=False,  # Whether drop the last incomplete batch
+    sampler=dict(type='DefaultSampler', shuffle=False),  # The type of data sampler
+    dataset=dict(  # Validation dataset config
+        type=dataset_type,  # Type of dataset
+        metainfo=dict(dataset_type='set5', task_name='sisr'),
+        data_root=data_root + '/Set5',  # Root path of data
+        data_prefix=dict(img='LRbicx2', gt='GTmod12'),  # Prefix of image path
+        pipeline=test_pipeline))
+test_dataloader = val_dataloader
+```
+
+</td>
+
+</tr>
+</thead>
+</table>
+
+### Evaluation settings
+
+We update evaluation settings in MMEdit 1.x. Important modifications are as following.
+
+- The evaluation field is split to `val_evaluator` and `test_evaluator`. The `interval` is moved to `train_cfg.val_interval`.
+- The metrics to evaluation are moved from `test_cfg` to `val_evaluator` and `test_evaluator`.
+
+<table class="docutils">
+<thead>
+  <tr>
+    <th> Original </th>
+    <th> New </th>
+<tbody>
+<tr>
+<td valign="top">
+
+```python
+train_cfg = None  # Training config
+test_cfg = dict(  # Test config
+    metrics=['PSNR'],  # Metrics used during testing
+    crop_border=scale)  # Crop border during evaluation
+
+evaluation = dict(  # The config to build the evaluation hook
+    interval=5000,  # Evaluation interval
+    save_image=True,  # Save images during evaluation
+    gpu_collect=True)  # Use gpu collect
+```
+
+</td>
+
+<td valign="top">
+
+```python
+val_evaluator = [
+    dict(type='MAE'),  # The name of metrics to evaluate
+    dict(type='PSNR', crop_border=scale),  # The name of metrics to evaluate
+    dict(type='SSIM', crop_border=scale),  # The name of metrics to evaluate
+]
+test_evaluator = val_evaluator
+
+train_cfg = dict(
+    type='IterBasedTrainLoop', max_iters=300000, val_interval=5000)  # Config of train loop type
+val_cfg = dict(type='ValLoop')  # The name of validation loop type
+test_cfg = dict(type='TestLoop')  # The name of test loop type
+```
+
+</td>
+
+</tr>
+</thead>
+</table>
+
+### Schedule settings
```

Comment:
You may also attach https://github.com/open-mmlab/mmengine/blob/main/docs/zh_cn/migration/migrate_param_scheduler_from_mmcv.md here.